packages/website/docs/crawler.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: DocSearch x Algolia Crawler
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+If you're not finding the answer to your question on this website, this page will help you. If you're still unsure, don't hesitate to send [your question to us][1] directly.
+
+You can also read our [Crawler FAQ](https://www.algolia.com/doc/tools/crawler/troubleshooting/faq), to understand how it behaves:
+
+- [One of my pages was not crawled](https://www.algolia.com/doc/tools/crawler/troubleshooting/faq/#one-of-my-pages-was-not-crawled)
+- [Why are my pages skipped?](https://www.algolia.com/doc/tools/crawler/troubleshooting/faq/#when-are-pages-skipped-or-ignored)
+
+For questions related to the DocSearch program, please see our [DocSearch program FAQ](/docs/DocSearch-program).
+
+## How often will you crawl my website?
+
+Crawls are scheduled at a random time once a week, you are also able to trigger new ones directly from [the Crawler interface][2]
+
+## Why do I have duplicate content in my results?
+
+This can happen when you have more than one URL pointing to the same content, for example with `./docs`, `./docs/` and `./docs/index.html`.
+
+We recommend configuring canonical URLs on your website, you can read more on the ["Consolidate duplicate URLs" guide by Google](https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls) and use our [`ignoreCanonicalTo`](https://www.algolia.com/doc/tools/crawler/apis/configuration/ignore-canonical-to/) option directly in your crawler config.
```

Comment:
ignoreCanonicalTo is the opposite advice of configuring canonical.
This prop is there to solves bad configuration, I would not add it to this doc

Reply:
Make sense, removed